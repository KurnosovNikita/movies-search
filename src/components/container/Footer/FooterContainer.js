import React from "react";
import TextBlock from '../../presentational/TextBlock';

class FooterContainer extends React.PureComponent {
    render() {
        const {footerClassName, contextClassName} = this.props;
        return (
          <footer className={footerClassName}>
              <div className={contextClassName}>
                  <TextBlock className="title" text="netflixroulette"/>
              </div>
          </footer>
        );
    }
}

export default FooterContainer;