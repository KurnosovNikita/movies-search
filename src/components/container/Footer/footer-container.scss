.footer {
    background-color: $hex-4;
    height: 50px;

    .title {
        font-size: 16px;
        color: $hex-3;
        padding: 15px 40px;
    }
}