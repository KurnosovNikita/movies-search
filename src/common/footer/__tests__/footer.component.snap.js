import React from 'react';
import {shallow} from 'enzyme';
import Footer from '../footer.component';

describe('Footer component', () => {
    let props;
    beforeEach(() => {
        props = {
            footerClassName: 'test',
            contextClassName: 'test',
        };
    });

    it('should render <Footer> correctly', () => {
        const component = shallow((<Footer {...props} />));

        expect(component).toMatchSnapshot();
    });
});
