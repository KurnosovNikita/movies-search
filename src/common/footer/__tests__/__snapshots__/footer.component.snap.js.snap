// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render <Footer> correctly 1`] = `
<footer
  className="test"
>
  <div
    className="test"
  >
    <Text
      className="title"
      text="netflixroulette"
    />
  </div>
</footer>
`;
